UnityProject/Assets/Strings/tips/GeneralTips.asset

```diff
@@ -0,0 +1,65 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 12b945c0690aa6045ae7abb331ebfe26, type: 3}
+  m_Name: GeneralTips
+  m_EditorClassIdentifier: 
+  strings:
+  - Alt-click while holding a health analyzer to get a full report on the body part
+    you're targeting plus the organs inside of it.
+  - All command roles and the clown have their own special rubber stamps
+  - Body parts can be dismembered and burnt using the right weapons and tools.
+  - Your lungs can bleed from the inside, forcing you to drown in your own blood.
+  - Security officers can heal a small amount of brute damage by eating donates.
+  - Lizards have thicker skins that allow them to have more chances against attacks
+    that might otherwise be deadily for other races.
+  - Play as a Jaintor as your first time. You have a clear goal and no one excepts
+    much from you.
+  - A Proto-Kinetic Accelerator (PKA) can destroy walls.
+  - Sometimes it's best to just kick back and relax at the bar or dorms and play
+    some cards or dungeon and dragons with your other crewmates.
+  - Killing your crewmates for no reason is the fastest way upset a mob -- and server
+    admins.
+  - As the quartermaster; It's your job to make sure that all departments are well
+    supplied. However, never be afraid of denying some requests. Seriously, Does
+    the clown really need 10 custome crates?
+  - Plants need water, nutrients and care. Keep an eye on them and make sure they
+    grow up healthy.
+  - You can use a crowbar to open airlocks and unpowered doors.
+  - Drinking too much alcohol is a garunteed way to build up toxin damage and destroy
+    your liver.
+  - Facehuggers cannot attatch themselves to your head if you wear a mask.
+  - Running makes you hungry faster.
+  - Hardsuits slow you down.
+  - In space, no one can hear you scream.
+  - Boxes automatically pick minnerals from the ground when pulled.
+  - Never take pills labaled as "Happy Pills".
+  - A crewmate who's comptent enough to do their job is an antagonist's worst nightmare.
```

Comment:
- A crewmate who's **competent** enough to do their job is an antagonist's worst nightmare.